docs/contributing/continuous-integration.rst

```diff
@@ -1,29 +1,44 @@
-.. _pulpcore: https://github.com/PulpQE/pulp-smash/tree/master/pulp_smash/tests/pulp3/pulpcore
-.. _file: https://github.com/PulpQE/pulp-smash/tree/master/pulp_smash/tests/pulp3/file
+.. _istqb: https://www.istqb.org/downloads/syllabi/foundation-level-syllabus.html
 .. _pulp-smash: https://github.com/PulpQE/pulp-smash/
-
-
 .. _continuous-integration:
 
 Continuous Integration
 ======================
 
-New code is highly encouraged to have basic unit tests that demonstrate its functionality. A Pull
-Request that has failing unit tests cannot be merged.
+Unit Tests
+----------
+New code is highly encouraged to have basic unit tests that demonstrate that
+units (function, method or class instance) are working correctly.
+A Pull Request that has failing unit tests cannot be merged.
+
+The unit tests for `pulpcore` are in `pulpcore/tests <https://github.com/pulp/pulp/tree/master/pulpcore/tests/units>`_.
```

Comment:
The link should be https://github.com/pulp/pulp/tree/master/pulpcore/tests/unit (no 's' at the end)

Reply:
Thanks.